crates/sncast/tests/helpers/devnet.rs

```diff
@@ -89,3 +102,111 @@ fn stop_devnet() {
         .spawn()
         .expect("Failed to kill devnet processes");
 }
+
+fn declare_and_deploy_contract(
+    contract_name: &str,
+    accounts_file: &str,
+    temp: &TempDir,
+) -> Contract {
+    let args = vec![
+        "--accounts-file",
+        accounts_file,
+        "--account",
+        "my_account",
+        "declare",
+        "--url",
+        URL,
+        "--contract-name",
+        contract_name,
+        "--max-fee",
+        "99999999999999999",
+        "--fee-token",
+        "strk",
+    ];
+
+    let snapbox = runner(&args).current_dir(temp.path());
+    let output = snapbox.assert().success();
+    let re_class_hash = Regex::new(r"class_hash:\s+(0x[a-fA-F0-9]+)").unwrap();
+
+    let class_hash = re_class_hash
+        .captures(output.as_stdout())
+        .and_then(|captures| captures.get(1))
+        .map(|match_| match_.as_str())
+        .expect("class_hash not found in the output");
+
+    let args = vec![
+        "--accounts-file",
+        accounts_file,
+        "--account",
+        "my_account",
+        "deploy",
+        "--url",
+        URL,
+        "--class-hash",
+        class_hash,
+        "--max-fee",
+        "99999999999999999",
+        "--fee-token",
+        "strk",
+    ];
+
+    let re_contract_address = Regex::new(r"contract_address:\s+(0x[a-fA-F0-9]+)").unwrap();
+
+    let snapbox = runner(&args).current_dir(temp.path());
+    let output = snapbox.assert().success();
+
+    let contract_address = re_contract_address
+        .captures(output.as_stdout())
+        .and_then(|captures| captures.get(1))
+        .map(|match_| match_.as_str())
+        .expect("contract_address not found in the output");
+
+    Contract {
+        class_hash: class_hash.to_string(),
+        contract_address: contract_address.to_string(),
+    }
+}
+
+#[must_use]
+pub fn prepare_accounts_file(temp: &TempDir) -> Utf8PathBuf {
```

Comment:
nit: Why not to use an account from the accounts file used in other tests?

Reply:
It's better if the account used in cast example purposes is independent from tests.

Reply:
But still it can be defined in a file somewhere int `tests/data/accounts` rather than directly in the code

Reply:
Sure, moved 👍